CHANGELOG.md

```diff
@@ -23,6 +23,8 @@ as necessary. Empty sections will not end in the release notes.
 ### Deprecations
 
 ### Fixes
+- Fixed potential index corruption when importing repositories into the new storage model that 
```

Comment:
nit: maybe "... when importing repositories with many keys ..." (just to be more precise)?